igvc_navigation/config/global_costmap_params.yaml

```diff
@@ -2,8 +2,8 @@ global_costmap:
     global_frame: odom
     robot_base_frame: base_link
     plugins:
-        - {name: wrapper_layer,     type: "wrapper_layer::WrapperLayer"}
```

Comment:
Can't we remove the files for  `wrapper_layer` at this point?

Reply:
Yeah we should be able to